mscore/instrwidget.h

```diff
@@ -31,6 +31,7 @@ class InstrumentGenre;
 
 enum class ListItemOp : char { KEEP, I_DELETE, ADD, UPDATE };
 enum { PART_LIST_ITEM = QTreeWidgetItem::UserType, STAFF_LIST_ITEM };
+enum PartiturSelectionType { PARENTS_SELECTED, CHILDREN_SELECTED, NONE_SELECTED };
```

Comment:
`enum class : char` would be preffered, I think? @dmitrio95 could confirm, possibly.

Reply:
done

Reply:
Um, no, I wouldn't confirm that :) I don't believe that this enumeration is used so frequently that storing it in one byte instead of 4 of 8 will save a lot of memory. So I would not bother making the underlying type as small as possible unless some measurements show that this really takes a lot of memory. Reverting that does not make much sense either though.

Reply:
Oh, sorry. I thought just having a plain `enum` instead of `enum class` is bad practice, though? For example, see: https://stackoverflow.com/a/12581154
I agree that it makes little difference in terms of memory usage.

Reply:
`enum class` is indeed generally better than plain old `enum`, that is true. I didn't notice that there was a plain `enum` here and thought that the point of your comment was that `: char` part which seems to be useful in much more limited number of cases. But you are totally correct in pointing to `enum` vs `enum class` difference, sorry for missing that part!

Reply:
It's now scoped. However, i'm still confused by the line above (`PART_LIST_ITEM, STAFF_LIST_ITEM`). Not sure of its purpose, it's used in constructing a `StaffListItem` (superclass of `QTreeWidgetItem`), but if I understand correctly it just passes `QTreeWidgetItem::UserType` in both cases. 
Removing the enum and using `QTreeWidgetItem::UserType` in the constructors does not cause any obvious problems. Any ideas?

Reply:
Ah, I think it does. However, that differentiation is never used. I'll leave it untouched for now.